### ISSUE TITLE
Constant logging every second about failed attempts to subscribe to events when rancher-server is down.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#10065]
Tested with rancher-server - v1.6-developement

Create a service - S1 with drain_timeout_ms set .
Create an LB service with service S1 as target.
Stop rancher server and start it after a while.

LB container logs the following set of logs every second.

```
10/10/2017 11:44:13 AMtime="2017-10-10T18:44:13Z" level=info msg="Initializing event router" workerCount=250
Unknown Date
10/10/2017 11:44:13 AMtime="2017-10-10T18:44:13Z" level=error msg="Error subscribing to events: dial tcp <ip>:8080: getsockopt: connection refused" subscribeUrl="ws://<ip>/v2-beta/subscribe?eventNames=target.drain&eventNames=ping"
10/10/2017 11:44:13 AMtime="2017-10-10T18:44:13Z" level=error msg="Exiting subscriber: dial tcp <ip>:8080: getsockopt: connection refused"
10/10/2017 11:44:14 AMtime="2017-10-10T18:44:14Z" level=info msg="Initializing event router" workerCount=250
Unknown Date
10/10/2017 11:44:14 AMtime="2017-10-10T18:44:14Z" level=error msg="Error subscribing to events: dial tcp <ip>:8080: getsockopt: connection refused" subscribeUrl="ws://<ip>/v2-beta/subscribe?eventNames=target.drain&eventNames=ping"
10/10/2017 11:44:14 AMtime="2017-10-10T18:44:14Z" level=error msg="Exiting subscriber: dial tcp <ip>8:8080: getsockopt: connection refused"
```

**Comment by prachidamle** [comment]
We can avoid logging the error statement twice:
 level=error msg="Exiting subscriber: dial tcp <ip>:8080: getsockopt: connection refused"

But the other two will come up from the event-subscriber for all services using it.

**Comment by deniseschannon** [comment]
Available with lb version v0.7.13

**Comment by sangeethah** [comment]
Tested with rancher/lb-service-haproxy:v0.7.13

```Exiting subscriber:``` line is not printed anymore 

```
10/13/2017 10:19:26 AMtime="2017-10-13T17:19:26Z" level=info msg="Initializing event router" workerCount=250
Unknown Date
10/13/2017 10:19:26 AMtime="2017-10-13T17:19:26Z" level=error msg="Error subscribing to events: dial tcp 130.211.225.50:8080: getsockopt: connection refused" subscribeUrl="ws://<ip>:8080/v2-beta/subscribe?eventNames=ping&eventNames=target.drain"
10/13/2017 10:19:27 AMtime="2017-10-13T17:19:27Z" level=info msg="Initializing event router" workerCount=250
Unknown Date
10/13/2017 10:19:27 AMtime="2017-10-13T17:19:27Z" level=error msg="Error subscribing to events: dial tcp 130.211.225.50:8080: getsockopt: connection refused" subscribeUrl="ws://<ip>:8080/v2-beta/subscribe?eventNames=ping&eventNames=target.drain"
```
